packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -293,7 +293,7 @@ export class List extends BaseComponent<IListProps, IListState> {
     let divProps = getNativeProps(this.props, divProperties);
 
     // assign list if no role
-    role = role || 'list';
+    role = (typeof role === 'undefined') ? 'list' : role;
```

Comment:
I guess I should actually respond on my own PR. Sorry my son has been sick :). I did the check because of the redefinition of undefined but yes it's overkill and I should have checked the codebase before I added it. I swapped it out in this PR for the standard way and I'll complete it that way.